### PR TITLE
Adds confidence interval display to cards and graph, adds ci display …

### PR DESCRIPTION
**Pull request by amygroshek**
…management controls

 * Changes for bar and line graph with sample data
 * Adds ci to card details, adds ci toggle to graph
 * Point angular-d3-graph to add-ci tag
 * tslint cleanup
 * Update lock file
 * Insert icon
 * Update labels and map card stats, add tooltip
 * Change tooltip format and add legend ci
 * Make bar graph tooltips and legend tooltips conditional
 * Update angular-d3-graph package
 * Update tooltip labels and content for graph
 * Fixup linting
 * CSS updates for card and legend mobile and tablet
 * Touch-up CSS and update package for graph
 * Update graph component
 * Adjustments for map location card display on mobile and tablet

### CODE REVIEW

**Comment by amygroshek** [comment]
@Lane These changes are already in development to stage for James. I've rebased in this branch to facilitate a PR. Let me know if it's easier some other way.

File: src/tsconfig.app.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
   },
   "include": [
     "../src/**/*",
-    "../node_modules/angular-d3-graph/module.ts"
+    "../node_modules/angular-d3-graph/module.ts",
+    "/Users/lougro/repos/angular-d3-graph/module.ts"
```
Comment (Lane) [inline src/tsconfig.app.json]: looks like an artifact from local testing, should be dropped before merge

File: src/tsconfig.spec.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
   "include": [
     "**/*.spec.ts",
     "**/*.d.ts",
-    "../node_modules/angular-d3-graph/module.ts"
+    "../node_modules/angular-d3-graph/module.ts",
+    "/Users/lougro/repos/angular-d3-graph/module.ts"
```
Comment (Lane) [inline src/tsconfig.spec.json]: one more local artifact

File: tsconfig.json [hunk 3]
```diff
@@ -18,5 +18,8 @@
       "es2017",
       "dom"
     ]
-  }
+  },
+  "include": [
+    "/Users/lougro/repos/angular-d3-graph/module.ts"
```
Comment (Lane) [inline tsconfig.json]: one more artifact from local testing

File: src/tsconfig.app.json (hunk 1 above)
Comment (amygroshek) [reply src/tsconfig.app.json]: Ah, good catch. I'll send a fixup straightaway.

**Pull request merged**